RUFAS/biophysical/animal/digestive_system/enteric_methane_calculator.py

```diff
@@ -0,0 +1,331 @@
+import numpy
+
+from RUFAS.biophysical.animal.digestive_system.methane_mitigation_calculator import MethaneMitigationCalculator
+from RUFAS.general_constants import GeneralConstants
+
+
+class EntericMethaneCalculator:
+    @staticmethod
+    def calf_methane(methane_model: str | None, body_weight: float) -> float:
```

Comment:
```suggestion
    def calculate_calf_methane(methane_model: str | None, body_weight: float) -> float:
```
These method names should start with verbs